composer.json

```diff
@@ -0,0 +1,8 @@
+{
+    "name": "cim/landr",
```

Comment:
I guess put it as demyx lol

Sent from my iPhone

> On Aug 6, 2016, at 07:40, Juho Jaakkola <notifications@github.com> wrote:
> 
> In composer.json:
> 
> > @@ -0,0 +1,8 @@
> > +{
> > +    "name": "cim/landr",
> Or perhaps even "manacim"? I'm really not sure which one you prefer. :)
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>